implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -34,14 +50,43 @@ private static void getDefaultFromClasspath(File propertyFile) {
         }
     }
 
+    private static PropertiesConfiguration getReferenceProperties(File propertyFile) {
```

Comment:
Only for similarity of those two methods (input: File, output: PropertiesConfiguration):

- `private static PropertiesConfiguration getReferenceProperties(File propertyFile)`
- `private static PropertiesConfiguration getTargetPropertiesFromJbbPath(File propertyFile)`